chat/message/src/MessageBanner/MessageBanner.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { Variant } from '@leafygreen-ui/banner';
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const baseStyles = css`
+  // Remove left wedge
+  &:before {
+    content: '';
+    background: transparent;
+  }
+
+  border-width: ${spacing[25]}px;
+  border-radius: ${spacing[600]}px;
+  border-left-color: unset;
+
+  max-width: fit-content;
+  padding: ${spacing[200]}px ${spacing[300]}px;
+`;
+
+/**
+ * These match the base border-color styles for each variant in the Banner component.
+ */
+const lightModeInfoMessageBannerStyles = css`
+  border-color: ${palette.blue.light2};
+`;
+const lightModeWarningMessageBannerStyles = css`
+  border-color: ${palette.yellow.light2};
+`;
+const lightModeDangerMessageBannerStyles = css`
+  border-color: ${palette.red.light2};
+`;
+const lightModeSuccessMessageBannerStyles = css`
+  border-color: ${palette.green.light2};
+`;
+const darkModeInfoMessageBannerStyles = css`
+  border-color: ${palette.blue.dark2};
+`;
+const darkModeWarningMessageBannerStyles = css`
+  border-color: ${palette.yellow.dark2};
+`;
+const darkModeDangerMessageBannerStyles = css`
+  border-color: ${palette.red.dark2};
+`;
+const darkModeSuccessMessageBannerStyles = css`
+  border-color: ${palette.green.dark2};
+`;
```

Comment:
Are these different from the base "Banner" component? Why?

Reply:
These are a re-implementation of the regular `border-color` in `@leafygreen-ui/banner`.

The original banner also has a different `border-left-color` set which shows up when you remove the wedge. Here's how `MessageBanner` looks without these styles applied:

<img width="250" alt="Screenshot 2024-04-02 at 11 54 23 AM" src="https://github.com/mongodb/leafygreen-ui/assets/15657698/9cd69e6c-c6c5-4a9d-b537-b0be051be56b">

Here it is without these styles but with `border-left-color: unset;`:

<img width="250" alt="Screenshot 2024-04-02 at 11 49 10 AM" src="https://github.com/mongodb/leafygreen-ui/assets/15657698/6f2593e0-bf2c-4a0c-a987-fa1d9d497f1c">

I didn't find a way to override the style so I re-implemented them instead. Obviously not very DRY! Any thoughts on a better way here?

Reply:
Honestly it doesn't look like [those border styles](https://github.com/mongodb/leafygreen-ui/blob/main/packages/banner/src/Banner/styles.ts#L32) are even used in `Banner`. Feel free to try and remove those, and if Chromatic passes, then great1

Reply:
Good call - I removed them from banner and there's no changes in Chromatic so should be good to go.